Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -2336,7 +2336,7 @@ RefPtr<API::Navigation> WebPageProxy::goToBackForwardItem(WebBackForwardListItem
                 loadParameters.navigationID = navigation->navigationID();
                 loadParameters.lockBackForwardList = LockBackForwardList::Yes;
                 frame->setHasPendingBackForwardItem(true);
-                frame->protectedProcess()->send(Messages::WebPage::LoadRequest(WTFMove(loadParameters)), webPageIDInMainFrameProcess());
+                frame->protectedProcess()->send(Messages::WebPage::LoadRequest(WTFMove(loadParameters)), webPageIDInProcess(frame->process()));
```

Comment:
It's a little strange to use two different getters for the same process in the same line.  Let's just save protectedProcess() into a local variable and use it twice.